pkg/asset/machines/master.go

```diff
@@ -253,12 +311,28 @@ func (m *Master) Files() []*asset.File {
 		files = append(files, m.UserDataFile)
 	}
 	files = append(files, m.MachineConfigFiles...)
+	// Add the host CRD so we don't block trying to create a type
+	// unknown to the cluster.
+	files = append(files, m.CRDFiles...)
+	// Hosts refer to secrets, so place the secrets before the hosts
+	// to avoid unnecessary reconciliation errors.
+	files = append(files, m.SecretFiles...)
+	// Machines are linked to hosts via the machineRef, so we create
+	// the hosts first to ensure if the operator starts trying to
+	// reconcile a machine it can pick up the related host.
+	files = append(files, m.HostFiles...)
 	files = append(files, m.MachineFiles...)
+
+	for _, file := range files {
+		logrus.Debugf("Master file %s %q", file.Filename, file.Data)
+	}
 	return files
 }
 
 // Load reads the asset files from disk.
 func (m *Master) Load(f asset.FileFetcher) (found bool, err error) {
+	var fileList []*asset.File
```

Comment:
define it right above where first required.

Reply:
ACK